lib/resolvers/post_query/posts.js

```diff
@@ -5,41 +5,58 @@ module.exports = async (parent, args) => {
   var isSortingExecuted = args.orderBy !== null;
 
   //Sorting List
-  if (isSortingExecuted) {
-    if (args.orderBy === 'id_ASC') {
-      sort = { _id: 1 };
-    } else if (args.orderBy === 'id_DESC') {
-      sort = { _id: -1 };
-    } else if (args.orderBy === 'text_ASC') {
-      sort = { text: 1 };
-    } else if (args.orderBy === 'text_DESC') {
-      sort = { text: -1 };
-    } else if (args.orderBy === 'title_ASC') {
-      sort = { title: 1 };
-    } else if (args.orderBy === 'title_DESC') {
-      sort = { title: -1 };
-    } else if (args.orderBy === 'createdAt_ASC') {
-      sort = { createdAt: 1 };
-    } else if (args.orderBy === 'createdAt_DESC') {
-      sort = { createdAt: -1 };
-    } else if (args.orderBy === 'imageUrl_ASC') {
-      sort = { imageUrl: 1 };
-    } else if (args.orderBy === 'imageUrl_DESC') {
-      sort = { imageUrl: -1 };
-    } else if (args.orderBy === 'videoUrl_ASC') {
-      sort = { videoUrl: 1 };
-    } else if (args.orderBy === 'videoUrl_DESC') {
-      sort = { videoUrl: -1 };
-    } else if (args.orderBy === 'likeCount_ASC') {
-      sort = { likeCount: 1 };
-    } else if (args.orderBy === 'likeCount_DESC') {
-      sort = { likeCount: -1 };
-    } else if (args.orderBy === 'commentCount_ASC') {
-      sort = { commentCount: 1 };
-    } else {
-      sort = { commentCount: -1 };
+    if (isSortingExecuted) {
```

Comment:
this is a bit over indented, this is the reason why the linting check is failing

Reply:
Make the changes and we can approve this PR.

Reply:
Done.